src/sub-blocks/LayoutItem/LayoutItem.tsx

```diff
@@ -15,7 +15,7 @@ import './LayoutItem.scss';
 const b = block('layout-item');
 
 const LayoutItem = ({
-    content: {links, buttons, ...content},
```

Comment:
why are we removing the buttons?

Reply:
Because we pass them to the `Content` component directly. We do not make any transformations with them like with links.